src/Pages/TermsAndConditions/index.tsx

```diff
@@ -6,34 +6,43 @@ const TermsAndConditions: React.FC = () => {
   return (
     <div className="tc">
       <Typography className="tc_title">Terms & Conditions</Typography>
-      <Typography className="tc_text">(Last updated: June 2021)</Typography>
+      <Typography className="tc_text">(Last updated: September 2023)</Typography>
       <Typography className="tc_text tc_paragraph">
         Please read these Terms carefully before participating on our auction platform. These Terms
         tell you who we are, what we offer and what to do if there is a problem and other important
         information. If you think that there is a mistake in these Terms, please contact us to
-        discuss at &nbsp;
-        <a className="tc_link" href="https://forum.gnosis.io/" rel="noreferrer" target="_blank">
-          forum.gnosis.io
+        discuss it at &nbsp;
+        <a
+          className="tc_link"
+          href="https://twitter.com/Fair_Auction"
+          rel="noreferrer"
+          target="_blank"
+        >
+          https://twitter.com/Fair_Auction
         </a>
         .
       </Typography>
       <Typography className="tc_text tc_paragraph tc_sub-title">
         Who we are and how to contact us
       </Typography>
       <Typography className="tc_text">
-        1. Fair Protcol is operated by GnosisDAO (“We”). We are an Ethereum based organisation
-        managed by community members. To contact us, please write to us via the GnosisDAO
-        subcategory of &nbsp;
-        <a className="tc_link" href="https://forum.gnosis.io/" rel="noreferrer" target="_blank">
-          forum.gnosis.io
+        1. Fair Protocol is operated by itself (“We”). We are an Ethereum-based organisation managed
+        by community members. To contact us, please message us on Twitter/X at &nbsp;
+        <a
+          className="tc_link"
+          href="https://twitter.com/Fair_Auction"
+          rel="noreferrer"
+          target="_blank"
+        >
+          https://twitter.com/Fair_Auction
         </a>
         .
       </Typography>
       <Typography className="tc_text">
-        2. We use open source products and developer materials from Gnosis Limited and affiliated
-        persons and entities(“Affiliates”) as well as other third parties to provide a Platform to
-        the Fair Protcol Protocol. The provision of the Platform and the Protocol is based solely on
-        our independent actions.
+        2. We use open-source products and developer materials from Gnosis Auction and affiliated
```

Comment:
Gnosis is still being used here

Reply:
That's fine as we are still utilising "Gnosis Auction" open source materials for dev